### PR TITLE
feature/PEER-331: Add pagination and footer 

### PR DESCRIPTION
**Pull request by alexeishulga**

### CODE REVIEW

File: app/containers/Questions/Content/Pagination/Pagination.tsx [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+const Pagination: React.FC = ({
+  page,
+  totalPages,
+  prevPage,
+  nextPage,
+  setPage,
+}: IProps): JSX.Element => {
+  const isDesktop450 = useMediaQuery('(min-width: 451px)');
+  if (totalPages <= 1) {
+    return '';
```
[the remark this answers is not in the document]
Comment (alexeishulga) [reply app/containers/Questions/Content/Pagination/Pagination.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1081/commits/f3b045c76d1eadc28c96be5d8ae4742fc227851a

File: app/containers/Questions/Content/Pagination/Pagination.tsx [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+                    ...styles.basicStyles,
+                    ...(page === el + 1 && styles.activeStyles),
+                  })}
+                  onClick={() => setPage(el + 1)}
+                  key={el}
+                >
+                  {el + 1}
+                </button>
+              ) : (
+                ''
```
[the remark this answers is not in the document]
Comment (alexeishulga) [reply app/containers/Questions/Content/Pagination/Pagination.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1081/commits/f3b045c76d1eadc28c96be5d8ae4742fc227851a

File: app/containers/Questions/Content/Pagination/Pagination.tsx [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
```
[the remark this answers is not in the document]
Comment (alexeishulga) [reply app/containers/Questions/Content/Pagination/Pagination.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1081/commits/f3b045c76d1eadc28c96be5d8ae4742fc227851a

File: app/containers/Questions/Content/Pagination/Pagination.tsx [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
```
[the remark this answers is not in the document]
Comment (alexeishulga) [reply app/containers/Questions/Content/Pagination/Pagination.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1081/commits/f3b045c76d1eadc28c96be5d8ae4742fc227851a

File: app/containers/Questions/Content/Pagination/Pagination.tsx [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
```
[the remark this answers is not in the document]
Comment (alexeishulga) [reply app/containers/Questions/Content/Pagination/Pagination.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1081/commits/f3b045c76d1eadc28c96be5d8ae4742fc227851a

File: app/containers/Questions/Content/Pagination/Pagination.tsx [hunk 6]
```diff
@@ -0,0 +1,291 @@
+import React from 'react';
+import { styles } from './Pagination.styled';
+import { css } from '@emotion/react';
+import prev from 'images/prev.svg?inline';
+import next from 'images/next.svg?inline';
+import useMediaQuery from '../../../../hooks/useMediaQuery';
+
+interface IProps {
```
[the remark this answers is not in the document]
Comment (alexeishulga) [reply app/containers/Questions/Content/Pagination/Pagination.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1081/commits/7f28a6d96b50018ee6be0920442b39678d3770e4

File: app/containers/Questions/Content/Pagination/Pagination.tsx [hunk 7]
```diff
@@ -0,0 +1,291 @@
...
+  page: number;
+  totalPages: number;
+  prevPage: () => void;
+  nextPage: () => void;
+  setPage: () => void;
+}
+
+const Pagination: React.FC = ({
```
[the remark this answers is not in the document]
Comment (alexeishulga) [reply app/containers/Questions/Content/Pagination/Pagination.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1081/commits/7f28a6d96b50018ee6be0920442b39678d3770e4

File: app/containers/Questions/Content/Pagination/Pagination.tsx [hunk 8]
```diff
@@ -0,0 +1,291 @@
...
+  const isDesktop450 = useMediaQuery('(min-width: 451px)');
+  if (totalPages <= 1) {
+    return '';
+  }
+  if (totalPages <= 5) {
+    return (
+      <>
+        <div className="df aic jcc">
+          {[...Array(totalPages).keys()].map(
+            (el, index) =>
```
[the remark this answers is not in the document]
Comment (alexeishulga) [reply app/containers/Questions/Content/Pagination/Pagination.tsx]: done: https://github.com/peeranha/peeranha-web/pull/1081/commits/7f28a6d96b50018ee6be0920442b39678d3770e4

**Review by Lampardks — APPROVED** [review]

**Pull request closed without merging**